lib/web/apiserver_test.go

```diff
@@ -8117,18 +8129,38 @@ func (r *testProxy) newClient(t *testing.T, opts ...roundtrip.ClientParam) *Test
 	return &TestWebClient{clt, t}
 }
 
+func makeAuthReqOverWS(ws *websocket.Conn, token string) error {
+	authReq, err := json.Marshal(struct {
+		Token string `json:"token"`
+	}{Token: token})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := ws.WriteMessage(websocket.TextMessage, authReq); err != nil {
```

Comment:
Do any of our clients care about the switch from `TextMessage` to `BinaryMessage`?

Reply:
Not that I can tell, it doesnt seem to be explicitly taken into account at least